ml/src/config.py

```
@@ -97,6 +121,48 @@ def _validate(cfg: dict) -> None:
+            isinstance(value, (list, tuple))
+            and len(value) == 2
+            and all(
+                isinstance(v, (int, float)) and not isinstance(v, bool) for v in value
+            )
+            and value[0] < value[1]
+        )
+        if not valid:
+            raise ValueError(
+                f"augmentation.{key} must be two ascending numbers "
+                f"[lower, upper], got {value!r}"
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)